modules/mobile/config/routes.rb

```diff
@@ -41,6 +41,7 @@
     put '/push/prefs/:endpoint_sid', to: 'push_notifications#set_pref'
     post '/push/send', to: 'push_notifications#send_notification'
     get '/user', to: 'users#show'
+    get '/user/contact_info', to: 'contact_info#show'
```

Comment:
I can never remember if we decided that our routes should use dashes or underscores. I thought we landed on dashes, but we have a mix, even for recently created endpoints. Like gender identity and preferrred name use underscores, but decision letters uses a dash. Maybe @jperk51 can provide some insight on this too.

Reply:
I think we maybe just talked about it but never actually followed through with it because it would require versioning all of our endpoints to sync them up, but yeah, we should stick to dashes. It's better for google search engines and average users aren't used to reading snaked phrases.